src/components/delete-movie-modal.jsx

```diff
@@ -0,0 +1,65 @@
+import { useMutation } from "@apollo/client";
+import { DELETE_MOVIE } from "../graphql/mutations/";
+import backArrow from "../images/back-arrow.svg";
+import redX from "../images/red-x-button.svg";
+import { Button } from "./button";
+import { Modal } from "./modal";
+
+export const DeleteMovieModal = ({
+  movieData,
+  closeModal,
+  currentUser,
+  refetchMovies,
+}) => {
+  const [deleteMovie, { data, loading, error }] = useMutation(DELETE_MOVIE);
+
+  const confirmDeleteMovie = (movieId) => {
```

Comment:
Works as intended. The modal pops up when delete is clicked and proceeds to ask if the user wants to delete the movie. Note: might be a refactor since outside scope but could display different text if the admin is the creator and wants to delete.

Reply:
Also it immediately removes from list which is good

Reply:
This was an easy fix since the other user's movies if they are an admin. In this case I just reversed the logic to check if the user id matched the movie author id first, if false then it displays the alternate text with the user's name